aspnetcore/fundamentals/native-aot.md

```diff
@@ -238,6 +238,22 @@ The `dotnet publish` command also compiles the source files and generates files
 
 [!INCLUDE[](~/fundamentals/aot/includes/aot_lib.md)]
 
+## Minimal APIs and JSON payloads
+
+The Minimal API framework is optimized for receiving and returning JSON payloads using <xref:System.Text.Json?displayProperty=fullName>. `System.Text.Json`:
+
+* Imposes compatibility requirements for JSON and native AOT.
+* Requires the use of the [`System.Text.Json` source generator](/dotnet/standard/serialization/system-text-json/source-generation).
+
+All types accepted as parameters to or returned from request delegates in Minimal APIs apps must be configured on a <xref:System.Text.Json.Serialization.JsonSerializerContext> that is registered via ASP.NET Core’s dependency injection:
```

Comment:
"All types that are transmitted as part of the HTTP body..."

You can have a parameter on the delegate that isn't bound to the body and does not to be serializable. For example, a query string parameter which is a rich object type which implements `IParsable<T>`.